README.adoc

```diff
@@ -18,7 +18,7 @@
 
 ## Status
 
-MicroProfile Config proposal!
+MicroProfile Config Framework
```

Comment:
I'd call it MP config **specification**. It's not a framework, because it provides no implementation. And I think that the word "specification" can be used even if it's not standardized by any organization. It's just a formal specification how implementations should behave.